farm/modeling/tokenization.py

```diff
@@ -292,11 +292,10 @@ def _words_to_tokens(words, word_offsets, tokenizer):
         elif len(tokens) == 0:
             tokens_word = tokenizer.tokenize(w)
         else:
-            try:
+            if type(tokenizer) == RobertaTokenizer:
```

Comment:
what about XLMRobertaTokenizer? What if we check if the \_\_name\_\_ string contains roberta to make it more general?